docs/release-notes.md

```diff
@@ -10,6 +10,98 @@ toc_max_heading_level: 2
 
 Welcome to Semgrep release notes. This document provides an overview of the changes, additions, and fixes made in different versions.
 
+## December 2021
+
+### Version 0.76.2
+
+#### Additions
+
+##### Support for Solidity
+
+Semgrep now provides experimental support for the Solidity programming language.
+
+#### Fixes
+
+##### Python
+
+Comprehension variables now have the correct scope, which means that a pattern like `[$X for $X in $ITERATOR]` now correctly matches [v for v in foo()]. ([#4260](https://github.com/returntocorp/semgrep/issues/4260))
+
+##### Semgrep reports relative file paths with `.semgrepignore`
+
+Previously, when you used Semgrep with .semgrepignore file, Semgrep reported targets with absolute instead of relative file paths. This issue has now been fixed. ([#4402](https://github.com/returntocorp/semgrep/pull/4402))
```

Comment:
```suggestion
Previously, when you used Semgrep with `.semgrepignore` file, Semgrep reported targets with absolute instead of relative file paths. This issue has now been fixed. ([#4402](https://github.com/returntocorp/semgrep/pull/4402))
```